src/main/resources/phrases_en_US.properties

```diff
@@ -75,6 +75,7 @@ QnVersion.intro=My current version is %s, \
 
 Reports.success=Done! FYI, the full log is [here](%s) (took me %[ms]s)
 Reports.failure=Oops, I failed. You can see the full log [here](%s) (spent %[ms]s)
+Reports.stop-fails=Sorry, I failed to stop the previous command, however it has next result: %s
```

Comment:
@mkordas We have only information that command ends with success/failure. And from last comment we cannot determine command (because it was `stop` command).